src/main/scala/com/amazon/deequ/suggestions/ConstraintSuggestionRunner.scala

```diff
@@ -70,13 +70,17 @@ class ConstraintSuggestionRunner {
       cacheInputs: Boolean,
       fileOutputOptions: ConstraintSuggestionFileOutputOptions,
       metricsRepositoryOptions: ConstraintSuggestionMetricsRepositoryOptions,
-      kllParameters: Option[KLLParameters])
+      kllWrapper: List[Any])
```

Comment:
Use the tuple from above the remove the casts below

Reply:
Sure. Will remove the cast.